### PR TITLE
Clean after multicall

### PR DESCRIPTION
**Pull request by juniset**
Some cleaning around multicalls and aligning with OZ.

### CODE REVIEW

File: test/utils/TransactionSender.py [hunk 1]
```diff
@@ -17,17 +17,8 @@ async def send_transaction(self, calls, signers, nonce=None, max_fee=0):
...
-        calldata = []
-        for i in range(len(calls)):
-            if len(calls[i]) != 3:
-                raise Exception("Invalid call parameters")
-            call = calls[i]
-            call_input = (call[0], get_selector_from_name(call[1]), len(calldata), len(call[2]))
-            call_inputs.append(call_input)
-            calldata.extend(call[2])
-            calls_with_selector.append((call[0], get_selector_from_name(call[1]), call[2]))
+        calls_with_selector = map(lambda call: (call[0], get_selector_from_name(call[1]), call[2]), calls)
```
Comment (delaaxe) [inline test/utils/TransactionSender.py:20]: `map(lambda: ...)` is an antipattern in Python. Always prefer list comprehensions:
```suggestion
        calls_with_selector = [(call[0], get_selector_from_name(call[1]), call[2]) for call in calls]
```

File: test/utils/TransactionSender.py [hunk 2]
```diff
@@ -37,7 +28,19 @@ async def send_transaction(self, calls, signers, nonce=None, max_fee=0):
...
+        return await self.account.__execute__(call_array, calldata, nonce).invoke(signature=signatures)
+
+def from_call_to_call_array(calls):
+    call_array = []
+    calldata = []
+    for i in range(len(calls)):
+        if len(calls[i]) != 3:
+            raise Exception("Invalid call parameters")
+        call = calls[i]
+        entry = (call[0], get_selector_from_name(call[1]), len(calldata), len(call[2]))
```
Comment (delaaxe) [inline test/utils/TransactionSender.py:40]: it's a very standard pattern to use `enumerate` when iterating over both indexes and items

```suggestion
    for i, call in enumerate(calls):
        assert len(call) == 3, "Invalid call parameters"
        entry = (call[0], get_selector_from_name(call[1]), len(calldata), len(call[2]))
```

File: test/utils/TransactionSender.py [hunk 3]
```diff
@@ -37,7 +28,19 @@ async def send_transaction(self, calls, signers, nonce=None, max_fee=0):
             else:    
                 signatures += list(signer.sign(message_hash))
 
-        return await self.account.__execute__(call_inputs, calldata, nonce).invoke(signature=signatures)
+        return await self.account.__execute__(call_array, calldata, nonce).invoke(signature=signatures)
+
+def from_call_to_call_array(calls):
+    call_array = []
```
Comment (delaaxe) [inline test/utils/TransactionSender.py:34]: nitpick: there are no real arrays in Python, `[]` represents lists

Comment (juniset) [reply test/utils/TransactionSender.py:34]: Yes but that's to match the naming used in the Cairo contract.

**Review by gergold — APPROVED** [review]

**Pull request merged**